smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -661,10 +665,63 @@ def locationHandler(evt) {
 
 def doDeviceSync(){
 	log.trace "Doing Hue Device Sync!"
+
+	// Check if state.updating failed to clear
+	if (state.lastUpdateStarted < (now() - 20 * 1000) && state.updating) {
+		state.updating = false
+		log.warn "state.updating failed to clear"
+	}
+
 	convertBulbListToMap()
 	poll()
 	ssdpSubscribe()
 	discoverBridges()
```

Comment:
Local polling only gives us changes when we query light status. As soon as IP changes for example, local polling will just stop until smartapp triggers a new poll with new IP.